data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -4,19 +4,19 @@ set -ex
 . /usr/local/bin/release-image.sh
 
 IRONIC_IMAGE=$(image_for ironic)
-IRONIC_INSPECTOR_IMAGE=$(image_for ironic-inspector)
 IPA_DOWNLOADER_IMAGE=$(image_for ironic-ipa-downloader)
 COREOS_DOWNLOADER_IMAGE=$(image_for ironic-machine-os-downloader || image_for ironic-rhcos-downloader)
 
-# This image is templated in via the installer pkg/asset/ignition/bootstrap/bootstrap.go
+# This image is templated in via the installer pkg/asset/ignition/bootstrap/common.go
 RHCOS_BOOT_IMAGE_URL="{{.BootImage}}"
+RHCOS_LIVE_IMAGE_URL="{{.PlatformData.BareMetal.ClusterOSImage}}"
 
 # This DHCP range is used by dnsmasq to serve DHCP to the cluster. If empty
 # dnsmasq will only serve TFTP, and DHCP will be disabled.
 DHCP_RANGE="{{.PlatformData.BareMetal.ProvisioningDHCPRange}}"
 
 # Used by ironic to allow ssh to running IPA instances
-IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"
+IRONIC_RAMDISK_SSH_KEY="$(echo -n "{{.SSHKey}}")"
```

Comment:
Why was the echo needed here?

Reply:
I saw instances where there was an added newline at the end of the key, so I had to add the echo.

Reply:
I wonder if Go templates have a function similar to Jinja's `| trim` filter.

Reply:
Ok, it seems unrelated to the rest of this change so I wonder if we should handle this in a separate bugfix PR (ideally with more details about exactly when/why this can be a problem)